package.json

```diff
@@ -8,7 +8,7 @@
     "node": ">=10.13.0"
   },
   "scripts": {
-    "start": "node server/server.js",
+    "start": "nodemon server/server.js",
```

Comment:
you can have two start scripts, one for node and one for nodemon so you don't have to keep on adding/deleting. i have a `npm run server-dev` which uses nodemon, but for deployment, i have a `npm run start` script that builds and runs node